### PR TITLE
Now work with several entity managers (Behat3)

### PR DESCRIPTION
**Pull request by gmorel**
Backport from @stephpy https://github.com/rezzza/alice-extension/pull/37 on Behat3

    - persist operation search for the good manager in registry
    - purge will purge each managers registered.

Keep up the good work

### CODE REVIEW

File: src/Rezzza/AliceExtension/Doctrine/ORMPersister.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Rezzza\AliceExtension\Doctrine;
+
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Nelmio\Alice\ORM\Doctrine;
+
+class ORMPersister extends Doctrine
```
Comment (tyx) [inline src/Rezzza/AliceExtension/Doctrine/ORMPersister.php]: In fact we should implements `Nelmio\Alice\ORMInterface` instead as we override the complete original behavior ?

**Comment by tyx** [comment]
looks nice, thanks :+1:

File: src/Rezzza/AliceExtension/Doctrine/ORMPersister.php (hunk 1 above)
Comment (gmorel) [inline src/Rezzza/AliceExtension/Doctrine/ORMPersister.php]: Good catch !

Unfortunately we have type hinted on `Nelmio\Alice\ORM\Doctrine`

```php
use Nelmio\Alice\ORM\Doctrine as ORMPersister;
```

[Adapter/ORM/ORMSubscriberFactory](https://github.com/rezzza/alice-extension/blob/49009145787559225a137c101bf898d76b950fb5/src/Rezzza/AliceExtension/Adapter/ORM/ORMSubscriberFactory.php#L7)
and here 
[Adapter/ORM/ORMEventSubscriber](https://github.com/rezzza/alice-extension/blob/master/src/Rezzza/AliceExtension/Adapter/ORM/ORMEventSubscriber.php#L11-L11)

Comment (tyx) [inline src/Rezzza/AliceExtension/Doctrine/ORMPersister.php]: Damned !

**Pull request merged**